core/webengine.py

```diff
@@ -37,10 +37,12 @@
 
 MOUSE_BACK_BUTTON = 8
 MOUSE_FORWARD_BUTTON = 16
+MOUSE_WHEEL_BUTTON = 4
 
 class BrowserView(QWebEngineView):
 
     translate_selected_text = QtCore.pyqtSignal(str)
+    url_hovered = ""
```

Comment:
No, this is area to write signal, please don't define class variable, you should define instance variable.

Looks next commit.